human_analysis/1_process_recount2_data.ipynb

```diff
@@ -48,12 +50,10 @@
     "import pandas as pd\n",
     "import numpy as np\n",
     "import rpy2\n",
-    "import seaborn as sns\n",
-    "from sklearn import preprocessing\n",
     "import pickle\n",
     "\n",
     "from ponyo import utils, train_vae_modules\n",
-    "from generic_expression_patterns_modules import process, calc\n",
+    "from generic_expression_patterns_modules import process\n",
     "\n",
     "np.random.seed(123)"
```

Comment:
Question: Why we need to set the random seed?

Reply:
This seed was added to make sure that results of this notebook were reproducible but I don't think I need it here since `ponyo` is has its own settings to ensure reproducible training results and now we're sampling all of the SRA data. So I think this can be removed